server/session/security_cookie.ts

```diff
@@ -69,5 +69,6 @@ export function getSecurityCookieOptions(
       return { isValid: true, path: '/' };
     },
     isSecure: config.cookie.secure,
+    sameSite: config.cookie.isSameSite || 'None',
```

Comment:
So if `isSameSite` is `false`, it will become `'None'` here, is it expected?

Reply:
yes expected, basically Kibana accepts `Strict`, `Lax` and `None`, while `false` is meant to be `None` in `kibana.yml` config to make it parity with the legacy config settings